apps/www/graphql/configs/codegen-dev.ts

```diff
@@ -0,0 +1,51 @@
+import type { CodegenConfig } from '@graphql-codegen/cli'
+
+import { loadEnvConfig } from '@next/env'
+
+loadEnvConfig(process.cwd())
+
+const config: CodegenConfig = {
+  overwrite: true,
+  generates: {
+    './src/graphql/cms/gql/': {
+      schema: './graphql/dato-cms.schema.graphql',
+      documents: ['./src/graphql/cms/**/*.{ts,tsx}'],
+      preset: 'client',
+      presetConfig: {
+        gqlTagName: 'gql',
+      },
+      config: {
+        scalars: {
+          ItemId: 'string',
+          IntType: 'number',
+          Date: 'string',
+          DateTime: 'string',
+        },
+      },
+      plugins: [],
+    },
+    './src/graphql/republik-api/gql/': {
+      schema: './graphql/republik-api.schema.graphql',
+      documents: ['./src/graphql/republik-api/**/*.{ts,tsx}'],
+      preset: 'client',
+      presetConfig: {
+        gqlTagName: 'gql',
+      },
+      config: {
+        scalars: {
+          ItemId: 'string',
+          IntType: 'number',
+          Date: 'string',
+          DateTime: 'string',
+        },
+      },
+      plugins: [],
+    },
+  },
+  // Ensure that the generated files are formatted directly to not cause diffs if there are no changes
+  hooks: {
+    afterOneFileWrite: ['prettier --write'],
```

Comment:
Not a new change but we could also just prettier-ignore the generated files? Better for performance?

Reply:
Added it to .eslintignore + .prettierignore